install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -2653,10 +2900,16 @@ clustermesh:
       # For GKE LoadBalancer, use annotation cloud.google.com/load-balancer-type: "Internal"
       # For EKS LoadBalancer, use annotation service.beta.kubernetes.io/aws-load-balancer-internal: 0.0.0.0/0
       annotations: {}
+      # @schema
+      # enum: [Local, Cluster]
+      # @schema
       # -- The externalTrafficPolicy of service used for apiserver access.
-      externalTrafficPolicy:
+      externalTrafficPolicy: Cluster
+      # @schema
+      # enum: [Local, Cluster]
+      # @schema
       # -- The internalTrafficPolicy of service used for apiserver access.
-      internalTrafficPolicy:
+      internalTrafficPolicy: Cluster
```

Comment:
There's a few changes here still that seem to be modifying the defaults.. though I don't know whether these should have the value `~` to make them null, or adopt these new specific defaults.

Reply:
^ Probably @cilium/sig-k8s would have a better understanding of this.

Reply:
I wanted to avoid another case of `type: [null, string]` here.

`externalTrafficPolicy` and `internalTrafficPolicy` are being defaulted to `Cluster` if not specified. ([See k8s API spec page](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#servicespec-v1-core))

> InternalTrafficPolicy describes how nodes distribute service traffic they receive on the ClusterIP. If set to "Local", the proxy will assume that pods only want to talk to endpoints of the service on the same node as the pod, dropping the traffic if there are no local endpoints. **The default value, "Cluster"**, uses the standard behavior of routing to all endpoints evenly (possibly modified by topology and other features).

> externalTrafficPolicy describes how nodes distribute service traffic they receive on one of the Service's "externally-facing" addresses (NodePorts, ExternalIPs, and LoadBalancer IPs). If set to "Local", the proxy will configure the service in a way that assumes that external load balancers will take care of balancing the service traffic between nodes, and so each node will deliver traffic only to the node-local endpoints of the service, without masquerading the client source IP. (Traffic mistakenly sent to a node with no endpoints will be dropped.) **The default value, "Cluster"**, uses the standard behavior of routing to all endpoints evenly (possibly modified by topology and other features). Note that traffic sent to an External IP or LoadBalancer IP from within the cluster will always get "Cluster" semantics, but clients sending to a NodePort from within the cluster may need to take traffic policy into account when picking a node.